onto_tool/bundle_schema.yaml

```diff
@@ -0,0 +1,321 @@
+  tools:
+    type: array
+    items:
+      type: object
+      required:
+        - name
+        - type
+        - arguments
+      properties:
+        name:
+          $ref: '#/definitions/non_empty_string'
+        type:
+          type: string
+          enum:
+            - Java
+        arguments:
+          type: array
+          items:
+            type: string
+          minLength: 2
+      allOf:
+        - if:
+            properties:
+              type:
+                const: Java
+          then:
+            properties:
+              jar:
+                $ref: '#/definitions/non_empty_string'
+            required:
+              - jar
+  actions:
+    type: array
+    items:
+      type: object
+      required:
+        - action
+      properties:
+        action:
+          type: string
+          enum:
+            - mkdir
+            - copy
+            - move
+            - transform
+            - markdown
+            - graph
+            - definedBy
+            - export
+            - sparql
+            - verify
+      allOf:
+        - if:
+            properties:
+              action:
+                const: verify
+          then:
+            properties:
+              source:
+                $ref: '#/definitions/non_empty_string'
+              includes:
+                $ref: '#/definitions/include_list'
+              type:
+                type: string
+                enum:
+                  - shacl
+                  - ask
+                  - select
+            allOf:
+              - if:
+                  properties:
+                    type:
+                      const: ask
+                then:
+                  allOf:
+                    - $ref: '#/definitions/multi_query_spec'
+                    - properties:
+                        expected:
+                          type: boolean
+                      required:
+                        - expected
+              - if:
+                  properties:
+                    type:
+                      const: select
+                then:
+                  allOf:
+                    - $ref: '#/definitions/multi_query_spec'
+                    - properties:
+                        target:
+                          $ref: '#/definitions/non_empty_string'
+              - if:
+                  properties:
+                    type:
+                      const: shacl
+                then:
+                  properties:
+                    target:
+                      $ref: '#/definitions/non_empty_string'
+                    shapes:
+                      type: object
+                      properties:
+                        source:
+                          $ref: '#/definitions/non_empty_string'
+                        includes:
+                          $ref: '#/definitions/include_list'
+                  required:
+                    - shapes
+            required:
+              - type
+              - source
```

Comment:
This is the schema for the new `verify` action. `type` and `source` are mandatory, the other options depend on `type`.